db/re/skill_db.txt

```diff
@@ -758,12 +758,12 @@
 522,0,0,0,0,0,0,10,1,no,0,0,0,weapon,0,0x0,		NJ_TOBIDOUGU,Shuriken Training
 523,9,6,1,-1,0x40,0,10,1,no,0,0,0,weapon,0,0x0,	NJ_SYURIKEN,Throw Shuriken
 524,9,8,1,-1,0x40,0,5,3,no,0,0,0,weapon,0,0x0,	NJ_KUNAI,Throw Kunai
-525,9,8,2,-1,0x2,0,5,-3:-3:-4:-4:-5,no,0,0x40000,0,weapon,0,0x0,	NJ_HUUMA,Throw Huuma Shuriken
+525,9,8,1,-1,0x2,0,5,-3:-3:-4:-4:-5,no,0,0x40000,0,weapon,0,0x0,	NJ_HUUMA,Throw Huuma Shuriken
```

Comment:
Throw Huuma Shuriken is no longer a ground unit skill, the splash range should be updated to 1:1:1:2:2 and the skill unit entries should be removed accordingly (there are no ground unit packets sent by kRO). The skill is similar to Judex or Adoramus for instance.

The skill also always sends a 
clif_skill_damage(src,bl,tick, status_get_amotion(src), 0, -30000, 1, skill_id, skill_lv, DMG_SKILL);

followed by the damage packet(s):
DE01 0D02 A4FBA500 90ACA300 EEE7350C 1C020000 E4020000 D08AFFFF 0100 0100 06
DE01 0D02 A4FBA500 52B99E00 EEE7350C 1C020000 E4020000 5C010000 FFFF 0300 08
DE01 0D02 A4FBA500 90ACA300 EEE7350C 1C020000 E4020000 00000000 FFFF 0100 08
83092E00A4FBA50001F4010000F4010000010000000000000000000000

Weird note, but if the attack misses on the target (which happened in the packet logs above), the div count is set to 1 and the 'miss' only appears once.